modular_skyrat/modules/customization/modules/clothing/shoes/shoes.dm

```diff
@@ -40,6 +40,10 @@
 	desc = "A fancy pair of high heels. Won't compensate for your below average height that much."
 	icon_state = "heels"
 
+/obj/item/clothing/shoes/high_heels/Initialize()
```

Comment:
undocumented change

Reply:
Oh, the hell? That's from another PR I also put up. That's my bad; I didn't intend for them to cross over.